pkg/users/local.js

```diff
@@ -57,13 +71,22 @@
             return Object.assign({}, account, details[i]);
         });
 
+    const groupsExtraInfo = sortGroups(
+        groups.map(group => {
+            const userlistPrimary = accountsInfo.filter(account => account.gid === group.gid).map(account => account.name);
+            const userlist = group.userlist.filter(el => el !== "");
+            return ({ ...group, userlistPrimary, userlist, members: userlist.length + userlistPrimary.length, isAdmin: admins.includes(group.name) });
+        })
```

Comment:
These 5 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17996-20221212-195940-2c392d36-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)